src/test/resources/net/sf/jabref/importer/fileformat/InspecSilverPlatterImporterTest.txt

```diff
@@ -1,30 +0,0 @@
-Record INSPEC
```

Comment:
Maybe, it should be renamed then. I wonder, why the test case do not fail when removing that file.